### PR TITLE
bump current code storage version for automation time and bump spec and transaction version

### PR DESCRIPTION
**Pull request by justinzhou93**

### CODE REVIEW

**Comment by justinzhou93** [comment]
forgot to bump the code storage version of automation time

**Comment by justinzhou93** [comment]
couldn't find any other variables to update, but double check me on the different levers here @rhuttman

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- switched scheduled time slots from minutes to hours. Migration used to clear all tasks in existing maps and queues.
+
+## [278] - 2022-03-14
```
Comment (irsal) [inline CHANGELOG.md:9]: why is this set for 3/14?

Comment (justinzhou93) [reply CHANGELOG.md]: merp

File: runtime/neumann/src/lib.rs [hunk 2]
```diff
@@ -165,10 +165,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("neumann"),
 	impl_name: create_runtime_str!("neumann"),
 	authoring_version: 1,
-	spec_version: 278,
+	spec_version: 280,
```
Comment (lreesby) [inline runtime/neumann/src/lib.rs:168]: why not `278` for neuman?

Comment (justinzhou93) [reply runtime/neumann/src/lib.rs]: whoops, not 280. should be 279. 278 is already in production. that was in the 1.2.8 version bump

**Review by lreesby — APPROVED** [review]
lgtm

**Review by irsal — APPROVED** [review]

**Pull request merged**